hbase-client/src/main/java/org/apache/hadoop/hbase/CacheEvictionStats.java

```diff
@@ -28,7 +29,7 @@ public final class CacheEvictionStats {
 
   private final long evictedBlocks;
   private final long maxCacheSize;
-  private final Map<byte[], Throwable> exceptions;
+  private final IdentityHashMap<byte[], Throwable> exceptions;
```

Comment:
I think this will be a bit dangerous if we will return this map to upper layer...